ci.sh

```diff
@@ -81,6 +81,9 @@ function run_tests {
       cargo miri run --manifest-path bench-cargo-miri/$BENCH/Cargo.toml
     done
   fi
+  # Ensure that `./miri bench` works on all relevant platforms.
+  # `slice-get-unchecked` is the least complex of all available benchmarks.
+  CARGO_EXTRA_FLAGS="" ./miri bench slice-get-unchecked
```

Comment:
This is the wrong place to put the check. This here will be run *for each target that we check*. It needs to either be gated by `-z "${MIRI_TEST_TARGET+exists}"` (to only run on the host) or be moved elsewhere.

Reply:
Also why the `CARGO_EXTRA_FLAGS=""`?

Reply:
> Also why the `CARGO_EXTRA_FLAGS=""`?

Since CARGO_EXTRA_FLAGS are set to `--locked`, I've cleared them to avoid issue with `--locked` being passed twice (which manifested itself [in CI run of ca60799 commit](https://github.com/rust-lang/miri/actions/runs/4896227247/jobs/8742786560#step:9:1533)). Is there a better way to do that? Perhaps I could document why these flags are cleared?

Reply:
Ah, that sounds like a bug in our miri script... maybe we just shouldn't have the `--locked` in the `install` command.